### ISSUE TITLE
WCAG本文:「2.3.1 3 回の閃光、又は閾値以下」の見直し

### ISSUE THREAD

**Issue by jidaikobo-shibata** [waic/wcag22#1588]
ID: `three-flashes-or-below-threshold`
[対象箇所](https://waic.jp/translations/WCAG22/#three-flashes-or-below-threshold)
[原文](https://www.w3.org/TR/WCAG22/#three-flashes-or-below-threshold)

**Comment by jidaikobo-shibata** [comment]
現行注記:
> この達成基準を満たさないコンテンツでは、利用者がそのウェブページ全体を使用できない恐れがあるため、ウェブページ上のすべてのコンテンツは他の達成基準を満たすために用いられているか否かにかかわらず、この達成基準を満たさなければならない。[適合要件 5: 非干渉](https://raw.githack.com/waic/wcag22/main/guidelines/index.html#cc5)を参照。

もんどさん案:
> この達成基準を満たさないコンテンツは、利用者がページ全体を利用する能力を損なう恐れがあるため、そのウェブページ上の全てのコンテンツは、他の達成基準を満たすために使用されているかどうかにかかわらず、この達成基準を満たさなければならない。適合要件 5: 非干渉を参照。

ん〜。「ページ全体」の改変が気になります。この訳注だけの言葉として現れるようなので、「ウェブページ全体」と違うニュアンスを込めているのかどうかが気になってしまいます。

また、この訳注はある種の定型句なので、PR前に、WG4の他の方のご意見を聞きたいです。

**Comment by caztcha** [comment]
原文

> Since any content that does not meet this success criterion can interfere with **a user's ability to use the whole page**, **all content on the web page** (whether it is used to meet other success criteria or not) must meet this success criterion. See [Conformance Requirement 5: Non-Interference](https://www.w3.org/TR/WCAG22/#cc5).


もんどさん案では、原文が「a user's ability to use the whole page」である (「a user's ability to use the whole **web** page」ではない) ことから、原文に忠実に「利用者がページ全体を利用する能力」と訳しているのですね。

その後の「all content on the web page」で「**その**ウェブページ上の全てのコンテンツ」と訳しているので、上述の「ページ全体」の「ページ」はつまり「ウェブページ」であることは、伝わるかなと思います。

**Comment by bakera** [comment]
いくつか気になったところがありまして、

- 原文の括弧がなくなっているような
- 冒頭の any が訳出できていないような
- 「利用者が…損なう」と読めて混乱するような
- 「そのウェブページ」はどのウェブページ? 原文的にはたぶん試験の対象になっている特定のページのイメージなのでしょうけれど。この the は訳出しなくても良いのでは?
- Non-Interferenceを「非干渉」としているのですから interfere は「干渉する」で良くないですか?

といったところで、たとえばこんな感じではどうでしょうか?

この達成基準を満たさないコンテンツはいかなるものであれ、利用者のページ全体を利用する能力に干渉する恐れがあるため、ウェブページ上の全てのコンテンツは (他の達成基準を満たすために使用されているかどうかにかかわらず)、この達成基準を満たさなければならない。適合要件 5: 非干渉を参照。

**Comment by momdo** [comment]
@jidaikobo-shibata 
修正をお願いします。

**Comment by jidaikobo-shibata** [comment]
PR修正しました。
